.vsts-ci.yml

```diff
@@ -0,0 +1,234 @@
+trigger:
+- release/2.1*
+- release/2.2*
+
+pr:
+- release/2.1*
+- release/2.2*
+
+jobs:
+- job: debug_osx
+  displayName: OSX Debug
+  pool:
+    vmImage: macOS-10.13
+  variables:
+    DOTNET_RUNTIME_ID: osx.10.12-x64
```

Comment:
@peterhuene This is the correct RID?

Reply:
That's the RID I use to run tests locally on my MBP.